openghg/store/_obscolumn.py

```diff
@@ -50,7 +51,7 @@ def read_file(
         platform: str = "satellite",
         source_format: str = "openghg",
         overwrite: bool = False,
-    ) -> Optional[Dict]:
+    ) -> dict:
```

Comment:
This should be fine since `BaseStore` has `from __future__ import annotations`, which allows 3.9+ style types: https://peps.python.org/pep-0585/#implementation. That PEP says there might be issues with mypy, but it seems to think the typing is ok for 3.8.